### PR TITLE
Jacq fixes2020edition

### PR DESCRIPTION
**Pull request by Thalpy**
## About The Pull Request

It's been year since Jacq was out and about, so I took a quick look over her code as she's coming up again. I think this should fix some of the problems!

Fixed incorrect chem reference
Fixed her song (old link was dead)
Fixed her teleportation
Fixed the canmove changes
Fixed the spooky spans

Adds runetext to the words she says
Updates her dialogue to reflect the year between her last visit

You can still date her!

## Why It's Good For The Game

Jacq works properly now and doesn't spam runtimes

## Changelog
:cl:
fix: fixes some bugs in jacqs code from edits to the codebase
/:cl:

### CODE REVIEW

File: code/modules/holiday/halloween/jacqueen.dm [hunk 1]
```diff
@@ -334,7 +334,7 @@
 			to_chat(C, "<span class='big warning'> You feel an overwhelming desire to [message]")
 		if(2)
 			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"If only you had a better upbringing, your ears are now full of my singing!\"</span>")
-			C.client.tgui_panel?.play_music("https://puu.sh/ExBbv.mp4")
+			C.client.tgui_panel?.play_music("https://fermi.u.catgirlsare.sexy/0iDOrQTa.mp4")
 		if(3)
 			visible_message("<b>[src]</b> waves their arms around, <span class='spooky'>\"You're cute little bumpkin, On your head is a pumpkin!\"</span>")
```
Comment (Detective-Google) [inline code/modules/holiday/halloween/jacqueen.dm:411]: you're **a** cute etc

Comment (Thalpy) [reply code/modules/holiday/halloween/jacqueen.dm:411]: I had to change it, heck!

File: tgui/packages/tgui-panel/styles/goon/chat-dark.scss [hunk 2]
```diff
@@ -892,6 +892,10 @@ em {
   font-style: italic;
 }
 
+.spooky {
```
Comment (LetterN) [inline tgui/packages/tgui-panel/styles/goon/chat-dark.scss:895]: please rebuild TGUI as this won't work

Comment (Thalpy) [reply tgui/packages/tgui-panel/styles/goon/chat-dark.scss:895]: Done! And thanks for pointing this out!

**Pull request merged**